subprojects/jacoco/src/integTest/groovy/org/gradle/testing/jacoco/plugins/JacocoPluginIntegrationTest.groovy

```diff
@@ -267,19 +257,17 @@ public class ThingTest {
         ":test" in nonSkippedTasks
         ":otherTests" in nonSkippedTasks
         file("build/jacoco/jacocoMerge.exec").exists()
-        htmlReport("build/reports/jacoco/mergedReport/html").totalCoverage() == 65
+        htmlReport("build/reports/jacoco/mergedReport/html").totalCoverage() == 64
```

Comment:
Thanks, could we change our sample project to make this the same across versions?